### ISSUE TITLE
F# syntax highlighting not working

### ISSUE THREAD

**Issue by FoggyFinder** [granicz/BlogEngine#5]
One of the listed _key features_ is "**Syntax highlighting for F# code blocks**"
But when I follow instructions from the ReadMe the code looks like this:

![_sample](https://user-images.githubusercontent.com/16510738/72664726-d15ef780-3a09-11ea-97af-2b1cab46ba0f.png)

So, there is no highlighting for code. Have I missed something?

**Comment by granicz** [comment]
You need to use `fsharp` as your format specifier in your triple-backtick block, so start your block with ` ```fsharp ` for the F# syntax highlighting to kick in.

**Comment by FoggyFinder** [comment]
That's understandable :-)

I attached screenshot from the default post of the repo:
[Using JS.Alert](https://github.com/granicz/BlogEngine/blob/master/src/Hosted/posts/2015-08-04-IntroducingTryWebSharper.md#using-jsalert)

That snippet does contain fsharp tag. 

Again, I just followed instructions from the ReadMe. Nothing more.

The html shows that code are in pre/code tag:

```
<pre><code class="language-fsharp">
```

and there are scripts:

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.14.2/highlight.min.js" type="text/javascript" charset="UTF-8"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.14.2/languages/fsharp.min.js" type="text/javascript" charset="UTF-8"></script>
```

**Comment by FoggyFinder** [comment]
Additional info:

There is no such problem if I use `dotnet run`

**Comment by granicz** [comment]
Can you give me the full URL you are viewing in the browser?

**Comment by FoggyFinder** [comment]
sure, it is

http://localhost:56001/blog/2015-08-04-IntroducingTryWebSharper.html

```
".tools/dotnet-serve.exe" -d build -p:56001 --default-extensions:.html -a 0.0.0.0
Starting server, serving build
Using default extensions .html
Listening on:
  http://localhost:56001
```

----

Odd. I had removed `build` folder and now everything is fine.

Anyway seems like `dotnet-serve` doesn't work correctly. I switched to the `master` branch and it is  still shows page from the other branch. Well, maybe it's my fault and I've done something wrong but still unexpected. 

Anyway, I'm closing the issue since I'm unable to reproduce.
